README.rst

```diff
@@ -1,109 +1,138 @@
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3889163.svg)](https://doi.org/10.5281/zenodo.3889163)
+========
+ HAL-CGP
+========
 
 
+.. image:: https://badge.fury.io/py/hal-cgp.svg
+    :target: https://badge.fury.io/py/hal-cgp
+	     
+.. image:: https://img.shields.io/badge/python-3.6-red.svg
+	   :target: https://www.python.org/downloads/release/python-369/
+.. image:: https://img.shields.io/badge/python-3.7-red.svg
+	   :target: https://www.python.org/
+.. image:: https://img.shields.io/badge/python-3.8-red.svg
+	   :target: https://www.python.org/
+.. image:: https://img.shields.io/badge/License-GPLv3-blue.svg
+	   :target: https://www.gnu.org/licenses/old-licenses/gpl-3.0.html
+.. image:: https://api.travis-ci.org/Happy-Algorithms-League/hal-cgp.svg?branch=master)
+	   :target: https://travis-ci.org/Happy-Algorithms-League/hal-cgp
+.. image:: http://www.mypy-lang.org/static/mypy_badge.svg
+	   :target: http://mypy-lang.org/
+.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
+	   :target: https://github.com/psf/black
+.. image:: https://coveralls.io/repos/github/Happy-Algorithms-League/python-gp/badge.svg?branch=master
+	   :target: https://coveralls.io/github/Happy-Algorithms-League/python-gp?branch=master
+
 Cartesian genetic programming (CGP) in pure Python.
 
 This library implements Cartesian genetic programming (e.g, Miller and Thomson, 2000; Miller, 2011) for symbolic regression in pure Python, targeting applications with expensive fitness evaluations. It provides Python data structures to represent and evolve two-dimensional directed graphs (genotype) that are translated into computational graphs (phenotype) implementing mathematical expressions. The computational graphs can be compiled as a Python functions, SymPy expressions (Meurer et al., 2017) or PyTorch modules (Paszke et al., 2017). The library currently implements an evolutionary algorithm, specifically (mu + lambda) evolution strategies adapted from Deb et al. (2002), to evolve a population of symbolic expressions in order to optimize an objective function.
 
-<div style="text-align:center"><img src="https://raw.githubusercontent.com/Happy-Algorithms-League/hal-cgp/master/cgp-sketch.png" alt="CGP Sketch" width="600"/></div>
+.. image:: ../cgp-sketch.png
+   :width: 600
+   :alt: CGP Sketch
+	 
+Figure from Jordan, Schmidt, Senn & Petrovici, "Evolving to learn: discovering interpretable plasticity rules for spiking networks", arxiv:2005.14149_.
 
-<sub>Figure from Jordan, Schmidt, Senn & Petrovici, "Evolving to learn: discovering interpretable plasticity rules for spiking networks", [ arxiv:2005.14149](https://arxiv.org/abs/2005.14149).</sub>
+.. _arxiv:2005.14149: https://arxiv.org/abs/2005.14149
 
+.. long-description-end
 
 A simple example of CGP applied to a symbolic regression problem can be found in `examples/example_evo_regression.py`.
 
-Searching numerical values for constants via local search
```

Comment:
somehow this bit disappeared. shouldn't this be included somewhere or do you plan to readd again in a separate PR?

Reply:
Yes, this will be added in a separate PR. I don't think that this should be in the README, at least not in this detailed version. Will add it into the docs in a later PR.